.travis.yml

```diff
@@ -1,20 +1,26 @@
 language: python
 
+dist: trusty
+
+sudo: false
+
 python:
   - "3.4"
   - "3.5"
+  - "3.6"
 
 env:
   global:
     secure: uH0xfi+u/FkddzkNdffIFMEv4edgHIE35no4E1nTZPCu8P3yraN9LOnfq+METCiHbFlC2O/6x32Azpg7x+jeOzfxdwCh7t5IpMg83cFV0wLZWqt940WV02LZTESjmGn5qFcJIp31pRZGxsr0S//TkfUrVNviwQ9j5kgztp7aCPfAiYYGrFEPaHuW8BiwZLVO6orAx3l/TPN3QeuF0nG0PVQiHmxbYhNTXVbjlYXRFW0swAfPBwit7yjPInqIpFbDD4slC5mhNS/EBffRE7kccPxLQAeyM69QVTkS0lHkgpEcir7NTLfKphA1uHoLtx+WutK8KwIQfzdhWOYzm0W8uVDUhAkL/p+bXT1SWMYGAtQTf6swatVjZgb0Hqb6xMXmhzMSb7vl/zhFTxkG+9TLF6edanj33j5I6wtxBUmB359Xaq9reNK1Yk6SuCAD56Yg90wHeEhgrxRURMWvjy1rV4vFPWRUx84wNdgHaTf89hiT9Vdp74eD8xqWEMgyOmgUfczJTqFcoNzfE1NKw9sycJkbuV4Sb/asjYGGadN+XTJBOAimjKj1xy3bbM2Aso6TuEMN15DE1Z9i3Ct9H9Lg9ThZvlE5EXJ3v+gP9XX81MpPyZ9pf84X1tNIiYwTzkg6JBwYiUVrf3P6M9pIKwvHSrSMos3Tn8tlIZHy4RTir/E=
 
-services:
-    - postgresql
-
 addons:
-    postgresql: "9.4"
+    postgresql: "9.6"
+    apt:
+        packages:
+            - postgresql-9.6-postgis-2.3
 
 before_script:
+    - psql -U postgres -c "create extension postgis"
```

Comment:
Wow.. this is necessary?? Should be handled by apt as a post-install hook..

Reply:
I'm just trying to see what worked for people in this thread: https://github.com/travis-ci/travis-ci/issues/6972